buildpack-run.sh

```diff
@@ -0,0 +1,5 @@
+# This is buildpack-run.sh
+git clone https://github.com/Macley-Kun/avaire.git
```

Comment:
Shouldn't this point to `https://github.com/avaire/avaire.git` instead of your fork?

Reply:
*kuch* yes. Changed it :P